src/charm.py

```diff
@@ -66,12 +66,12 @@ def version(self) -> Optional[str]:
 
 
 class AlertmanagerCharm(CharmBase):
-    _container_name: str = "alertmanager"  # automatically determined from charm name
+    _container_name = "alertmanager"  # automatically determined from charm name
```

Comment:
you could probably shorten this: `_conainer_name = _layer_name = _service_name = "alertmanager"`

Reply:
Done